src/Universum.hs

```diff
@@ -5,7 +5,7 @@
 -- module to replace default ugly 'Prelude' with better one.
 --
 -- @
---     {-# LANGUAGE NoImplicitPrelude #-}
+--     {-\# LANGUAGE NoImplicitPrelude \#-}
```

Comment:
Thank you for finally fixing this documentation issue! I have no idea how `#` is rendered in Haddock...

Reply:
I also didn't find info about what `#` does there..